src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -160,6 +169,19 @@ private void readFromNBT(NBTTagCompound compound)
             }
         }
 
+        // Restore colony achievements
+        final NBTTagList achievementTagList = compound.getTagList(TAG_ACHIEVEMENT_LIST, NBT.TAG_COMPOUND);
+        this.colonyAchievements = new ArrayList<>();
+        for (int i = 0; i < achievementTagList.tagCount(); ++i)
+        {
+            final NBTTagCompound achievementCompound = achievementTagList.getCompoundTagAt(i);
+            final String achievementKey = achievementCompound.getString(TAG_ACHIEVEMENT);
+            final StatBase statBase = StatList.getOneShotStat(achievementKey);
+            if (statBase instanceof Achievement) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)